docker/Dockerfile

```
@@ -0,0 +1,73 @@
+COPY tsconfig.json /home/app
+WORKDIR /home/app
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
+RUN npx pnpm run build:view
+
+
+FROM swift:6.0.3-noble AS server-build
+
+ARG VERSION
+ARG GIT_COMMIT
+
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)